test/Chainweb/Test/Pact/Utils.hs

```diff
@@ -341,16 +226,30 @@ pDecimal = PLiteral . LDecimal
 pBool :: Bool -> PactValue
 pBool = PLiteral . LBool
 
+
+-- ----------------------------------------------------------------------- --
+-- Capability helpers
+
 -- | Cap smart constructor.
 mkCapability :: ModuleName -> Text -> [PactValue] -> SigCapability
 mkCapability mn cap args = SigCapability (QualifiedName mn cap def) args
 
+-- | Convenience to make caps like TRANSFER, GAS etc.
+mkCoinCap :: Text -> [PactValue] -> SigCapability
+mkCoinCap n = mkCapability "coin" n
+
 mkTransferCap :: Text -> Text -> Decimal -> SigCapability
-mkTransferCap sender receiver amount = mkCapability "coin" "TRANSFER"
+mkTransferCap sender receiver amount = mkCoinCap "TRANSFER"
   [ pString sender, pString receiver, pDecimal amount ]
 
 gasCap :: SigCapability
-gasCap = mkCapability "coin" "GAS" []
+gasCap = mkCoinCap "GAS" []
```

Comment:
```suggestion
mkGasCap = mkCoinCap "GAS" []
```
(optional) just to make name consistent with `mkTransferCap` etc

Reply:
👍